hoff.nix

```
@@ -0,0 +1,86 @@
+{ lib, haskellPackages, nix-gitignore, git, coreutils, openssh }:
+  haskellPackages.mkDerivation {
+    pname = "hoff";
+    version = "0.25";
```

We have x.y.z version tags, so `version` should probably match the next tag:

```suggestion
    version = "0.25.0";
```